scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This script needs the full bash standard boilerplate.
See other scripts in this project, e.g.,
https://github.com/edulinq/quizgen/blob/main/scripts/profile_tests.sh